specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json

```diff
@@ -1246,7 +1246,282 @@
+          },
+          {
+            "$ref": "./apimproducts.json#/parameters/ProductIdParameter"
+          },
+          {
+            "$ref": "#/parameters/TagIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Empty response body, ETag header entity state version.",
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tag_GetByProduct",
+        "description": "Get tag associated with the Product.",
+        "x-ms-examples": {
+          "ApiManagementGetProductTag": {
+            "$ref": "./examples/ApiManagementGetProductTag.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimproducts.json#/parameters/ProductIdParameter"
+          },
+          {
+            "$ref": "#/parameters/TagIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Gets the details of the tag specified by its identifier.",
+            "schema": {
+              "$ref": "./apimtags.json#/definitions/TagContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tag_AssignToProduct",
```

Comment:
`PUT` operations must usually be named as `Tag_create` or `Tag_Update`

Reply:
The PR was signed off by ARM https://github.com/Azure/azure-rest-api-specs/pull/2257. We are just moving operation into different file, as it caused the Documentation CI to break